util/stats/src/lib.rs

```diff
@@ -121,7 +133,19 @@ impl<T: Ord + Copy + ::std::fmt::Display> Histogram<T>
 
 #[cfg(test)]
 mod tests {
-	use super::Histogram;
+	use super::*;
+
+	#[test]
+	fn check_corpus() {
+		let corpus = Corpus::from(vec![1, 2, 3, 4, 5, 6, 7, 8, 9, 10]);
+		assert_eq!(corpus.percentile(0), None);
+		assert_eq!(corpus.percentile(1), None);
+		assert_eq!(corpus.percentile(101), Some(&10));
+		assert_eq!(corpus.percentile(100), Some(&10));
+		assert_eq!(corpus.percentile(50), Some(&5));
```

Comment:
I think for a bigger data set this should be the same as median.

Reply:
We have even number of elements here.